<!DOCTYPE html>
<html lang="en" dir="auto">

<head><meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>Research | LIAO RESEARCH GROUP @ HKUST</title>
<meta name="keywords" content="">
<meta name="description" content="Our lab employs a multidisciplinary approach to study the relationship between gene circuits and their dynamical properties in bacteria: quantitative fluorescence microscopy and microfluidics to capture temporal dynamics and cellular heterogeneities, molecular genetics to dissect biological mechanisms, machine learning to uncover patterns, and mathematical modeling to rebuild biological systems and predict experimental outcomes. Our current research topics fall into the following general categories:
Temporal signal processing
 Much like consciousness emerges from interactions within networks of neurons, gene regulatory networks can give rise to novel emergent properties that cannot be understood or predicted from their individual components in isolation. Even simple circuits comprising just two or three genes can generate sophisticated temporal behaviors in bacteria, including stochastic pulsing, circadian oscillations, fold-detection, noise filtering, and many others. We are investigating how various circuit attributes—such as network topology, reaction kinetics, and stoichiometry—shape temporal outputs and signal processing capabilities of gene circuits and to what extent these properties can be tuned experimentally.">
<meta name="author" content="">
<link rel="canonical" href="https://liaolab-qbio.com/research/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.79b82956f1518785e7fa631576ce76efb7c78811a945140cfcf342ed54f084b1.css" integrity="sha256-ebgpVvFRh4Xn&#43;mMVds5277fHiBGpRRQM/PNC7VTwhLE=" rel="preload stylesheet" as="style">
<link rel="icon" href="https://liaolab-qbio.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://liaolab-qbio.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://liaolab-qbio.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://liaolab-qbio.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://liaolab-qbio.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<link rel="alternate" hreflang="en" href="https://liaolab-qbio.com/research/">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
</noscript><meta property="og:url" content="https://liaolab-qbio.com/research/">
  <meta property="og:site_name" content="LIAO RESEARCH GROUP @ HKUST">
  <meta property="og:title" content="Research">
  <meta property="og:description" content="Our lab employs a multidisciplinary approach to study the relationship between gene circuits and their dynamical properties in bacteria: quantitative fluorescence microscopy and microfluidics to capture temporal dynamics and cellular heterogeneities, molecular genetics to dissect biological mechanisms, machine learning to uncover patterns, and mathematical modeling to rebuild biological systems and predict experimental outcomes. Our current research topics fall into the following general categories:
Temporal signal processing Much like consciousness emerges from interactions within networks of neurons, gene regulatory networks can give rise to novel emergent properties that cannot be understood or predicted from their individual components in isolation. Even simple circuits comprising just two or three genes can generate sophisticated temporal behaviors in bacteria, including stochastic pulsing, circadian oscillations, fold-detection, noise filtering, and many others. We are investigating how various circuit attributes—such as network topology, reaction kinetics, and stoichiometry—shape temporal outputs and signal processing capabilities of gene circuits and to what extent these properties can be tuned experimentally.">
  <meta property="og:locale" content="en-us">
  <meta property="og:type" content="article">
    <meta property="article:published_time" content="2025-02-04T16:19:08+08:00">
    <meta property="article:modified_time" content="2025-02-04T16:19:08+08:00">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Research">
<meta name="twitter:description" content="Our lab employs a multidisciplinary approach to study the relationship between gene circuits and their dynamical properties in bacteria: quantitative fluorescence microscopy and microfluidics to capture temporal dynamics and cellular heterogeneities, molecular genetics to dissect biological mechanisms, machine learning to uncover patterns, and mathematical modeling to rebuild biological systems and predict experimental outcomes. Our current research topics fall into the following general categories:
Temporal signal processing
 Much like consciousness emerges from interactions within networks of neurons, gene regulatory networks can give rise to novel emergent properties that cannot be understood or predicted from their individual components in isolation. Even simple circuits comprising just two or three genes can generate sophisticated temporal behaviors in bacteria, including stochastic pulsing, circadian oscillations, fold-detection, noise filtering, and many others. We are investigating how various circuit attributes—such as network topology, reaction kinetics, and stoichiometry—shape temporal outputs and signal processing capabilities of gene circuits and to what extent these properties can be tuned experimentally.">


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [
    {
      "@type": "ListItem",
      "position":  1 ,
      "name": "Research",
      "item": "https://liaolab-qbio.com/research/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "Research",
  "name": "Research",
  "description": "Our lab employs a multidisciplinary approach to study the relationship between gene circuits and their dynamical properties in bacteria: quantitative fluorescence microscopy and microfluidics to capture temporal dynamics and cellular heterogeneities, molecular genetics to dissect biological mechanisms, machine learning to uncover patterns, and mathematical modeling to rebuild biological systems and predict experimental outcomes. Our current research topics fall into the following general categories:\nTemporal signal processing Much like consciousness emerges from interactions within networks of neurons, gene regulatory networks can give rise to novel emergent properties that cannot be understood or predicted from their individual components in isolation. Even simple circuits comprising just two or three genes can generate sophisticated temporal behaviors in bacteria, including stochastic pulsing, circadian oscillations, fold-detection, noise filtering, and many others. We are investigating how various circuit attributes—such as network topology, reaction kinetics, and stoichiometry—shape temporal outputs and signal processing capabilities of gene circuits and to what extent these properties can be tuned experimentally.\n",
  "keywords": [
    
  ],
  "articleBody": "Our lab employs a multidisciplinary approach to study the relationship between gene circuits and their dynamical properties in bacteria: quantitative fluorescence microscopy and microfluidics to capture temporal dynamics and cellular heterogeneities, molecular genetics to dissect biological mechanisms, machine learning to uncover patterns, and mathematical modeling to rebuild biological systems and predict experimental outcomes. Our current research topics fall into the following general categories:\nTemporal signal processing Much like consciousness emerges from interactions within networks of neurons, gene regulatory networks can give rise to novel emergent properties that cannot be understood or predicted from their individual components in isolation. Even simple circuits comprising just two or three genes can generate sophisticated temporal behaviors in bacteria, including stochastic pulsing, circadian oscillations, fold-detection, noise filtering, and many others. We are investigating how various circuit attributes—such as network topology, reaction kinetics, and stoichiometry—shape temporal outputs and signal processing capabilities of gene circuits and to what extent these properties can be tuned experimentally.\nPhenotypic heterogeneity There is a prevailing notion in biology that phenotypes are determined by the interplay between genetics and environmental factors. Yet, bacterial cells with identical genomes and external conditions could exhibit strikingly different behaviors. A particularly relevant example is the spontaneous and transient emergence of antibiotic-tolerant cells within a genetically uniform population. We would like to uncover the gene regulatory circuits that are prone to drive phenotypic heterogeneities, unravel the key dynamical properties that enable stochastic cell-state transitions, and elucidate their physiological roles in the context of bacterial stress adaptation.\nNew methodologies We are developing advanced microfluidic and imaging platforms to uncover and characterize dynamic processes in single bacterial cells and microbial communities. A key objective is to simultaneously achieve high temporal resolution, single-cell sensitivity, and high throughput, such that transient and heterogeneous events are faithfully and unambiguously captured under well-defined external conditions. In parallel, we are working on statistical and machine learning models for inferential and predictive analyses of circuit-driven gene expression dynamics, offering a scalable alternative to traditional differential equation-based approaches.\nWe are grateful for the trust and generosity of our funding agencies, whose support allows us to pursue cutting-edge research with curiosity and purpose.\n",
  "wordCount" : "356",
  "inLanguage": "en",
  "datePublished": "2025-02-04T16:19:08+08:00",
  "dateModified": "2025-02-04T16:19:08+08:00",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://liaolab-qbio.com/research/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "LIAO RESEARCH GROUP @ HKUST",
    "logo": {
      "@type": "ImageObject",
      "url": "https://liaolab-qbio.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://liaolab-qbio.com/" accesskey="h" title="LIAO RESEARCH GROUP @ HKUST (Alt + H)">
                <img src="https://liaolab-qbio.com/UST_logo_2_resized.jpg" alt="" aria-label="logo"
                    height="43">LIAO RESEARCH GROUP @ HKUST</a>
            <div class="logo-switches">
            </div>
        </div>
        <ul id="menu">
            <li>
                <a href="https://liaolab-qbio.com/" title="Home">
                    <span>Home</span>
                </a>
            </li>
            <li>
                <a href="https://liaolab-qbio.com/research/" title="Research">
                    <span class="active">Research</span>
                </a>
            </li>
            <li>
                <a href="https://liaolab-qbio.com/people/" title="People">
                    <span>People</span>
                </a>
            </li>
            <li>
                <a href="https://liaolab-qbio.com/publications/" title="Publications">
                    <span>Publications</span>
                </a>
            </li>
            <li>
                <a href="https://liaolab-qbio.com/teaching/" title="Teaching">
                    <span>Teaching</span>
                </a>
            </li>
        </ul>
    </nav>
</header>
<main class="main">

<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title entry-hint-parent">
      Research
    </h1>
  </header> 
  <div class="post-content"><p>Our lab employs a multidisciplinary approach to study the relationship between gene circuits and their dynamical properties in bacteria: quantitative fluorescence microscopy and microfluidics to capture temporal dynamics and cellular heterogeneities, molecular genetics to dissect biological mechanisms, machine learning to uncover patterns, and mathematical modeling to rebuild biological systems and predict experimental outcomes. Our current research topics fall into the following general categories:</p>
<h3 id="temporal-signal-processing">Temporal signal processing<a hidden class="anchor" aria-hidden="true" href="#temporal-signal-processing">#</a></h3>
<p><img alt="temporal_signal_processing" loading="lazy" src="/temporal_signal_processing.jpg#floatleft_style_1"> Much like consciousness emerges from interactions within networks of neurons, gene regulatory networks can give rise to novel emergent properties that cannot be understood or predicted from their individual components in isolation. Even simple circuits comprising just two or three genes can generate sophisticated temporal behaviors in bacteria, including stochastic pulsing, circadian oscillations, fold-detection, noise filtering, and many others. We are investigating how various circuit attributes—such as network topology, reaction kinetics, and stoichiometry—shape temporal outputs and signal processing capabilities of gene circuits and to what extent these properties can be tuned experimentally.</p>
<h3 id="phenotypic-heterogeneity">Phenotypic heterogeneity<a hidden class="anchor" aria-hidden="true" href="#phenotypic-heterogeneity">#</a></h3>
<p><img alt="Phenotypic_heterogeneity" loading="lazy" src="/Phenotypic_heterogeneity.jpg#floatleft_style_1"> There is a prevailing notion in biology that phenotypes are determined by the interplay between genetics and environmental factors. Yet, bacterial cells with identical genomes and external conditions could exhibit strikingly different behaviors. A particularly relevant example is the spontaneous and transient emergence of antibiotic-tolerant cells within a genetically uniform population. We would like to uncover the gene regulatory circuits that are prone to drive phenotypic heterogeneities, unravel the key dynamical properties that enable stochastic cell-state transitions, and elucidate their physiological roles in the context of bacterial stress adaptation.</p>
<h3 id="new-methodologies">New methodologies<a hidden class="anchor" aria-hidden="true" href="#new-methodologies">#</a></h3>
<p><img alt="Methodology" loading="lazy" src="/Methodology.jpg#floatleft_style_1"> We are developing advanced microfluidic and imaging platforms to uncover and characterize dynamic processes in single bacterial cells and microbial communities. A key objective is to simultaneously achieve high temporal resolution, single-cell sensitivity, and high throughput, such that transient and heterogeneous events are faithfully and unambiguously captured under well-defined external conditions. In parallel, we are working on statistical and machine learning models for inferential and predictive analyses of circuit-driven gene expression dynamics, offering a scalable alternative to traditional differential equation-based approaches.</p>
<hr>
<p><img alt="funding_resources_1" loading="lazy" src="/funding_resources_1.jpg#floatright_style_2"> <p class="custom-font-size-1">We are grateful for the trust and generosity of our funding agencies, whose support allows us to pursue cutting-edge research with curiosity and purpose.</p></p>


  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
        <span>&copy; 2025 <a href="https://liaolab-qbio.com/">LIAO RESEARCH GROUP @ HKUST</a></span>


</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
</body>

</html>
